web/packages/teleterm/src/ui/ConnectMyComputer/NavigationMenu.tsx

```diff
@@ -69,22 +75,25 @@ export function NavigationMenu(props: NavigationMenuProps) {
 
   return (
     <>
-      <NavigationMenuIcon onClick={togglePopover} ref={iconRef} />
-      <Popover
-        open={isPopoverOpened}
+      <NavigationMenuIcon onClick={toggleMenu} ref={iconRef} />
+      <Menu
+        getContentAnchorEl={null}
+        open={isMenuOpened}
         anchorEl={iconRef.current}
         anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
         transformOrigin={{ vertical: 'top', horizontal: 'right' }}
-        onClose={() => setIsPopoverOpened(false)}
+        onClose={() => setIsMenuOpened(false)}
       >
-        <Container width="200px">
-          <ListItem onClick={openSetupDocument}>Connect computer</ListItem>
-        </Container>
-      </Popover>
+        {isSetupDoneAttempt.status === 'success' && (
```

Comment:
What should happen if this fails? At the moment it seems like we'll just hide the menu items. What should we do to make the app handle failure well here?

Reply:
I fixed that by showing a banner with an error on the status page and allowing the user to restart the setup process.